spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/http/RestTemplateTransportClientFactory.java

```diff
@@ -80,6 +123,22 @@ private RestTemplate restTemplate(String serviceUrl) {
 		return restTemplate;
 	}
 
+	private RestTemplate restTemplate() {
+		if (this.sslContext.isPresent()) {
+			SSLContext sslContext = this.sslContext.get();
+			// TODO: Using HttpClient here - maybe this should be configurable?
```

Comment:
What do we do about this?

Reply:
Could it be a bean?  That way you can provide your own?